web/src/components/Header/Header.tsx

```diff
@@ -1,64 +1,33 @@
-import {QuestionCircleOutlined} from '@ant-design/icons';
 import {useTour} from '@reactour/tour';
+import {Typography} from 'antd';
 import Logo from 'assets/Logo.svg';
-import {DOCUMENTATION_URL, GITHUB_URL} from 'constants/Common.constants';
-import {FC} from 'react';
+import {FC, useState} from 'react';
 import {Link, useLocation} from 'react-router-dom';
-import HomeAnalyticsService from 'services/Analytics/HomeAnalytics.service';
+import HomeAnalyticsService from '../../services/Analytics/HomeAnalytics.service';
 import * as S from './Header.styled';
+import './index.css';
+import {HeaderMenu} from './HeaderMenu';
 
 const {onGuidedTourClick} = HomeAnalyticsService;
 
 const Header: FC = () => {
   const {pathname} = useLocation();
   const {setIsOpen} = useTour();
-
-  const handleGuidedTourCLick = () => {
-    setIsOpen(true);
-    onGuidedTourClick();
-  };
-
+  const [visible, setVisible] = useState(false);
   return (
     <S.Header>
       <Link to="/">
         <S.TitleText>
           <img alt="tracetest_log" data-cy="logo" src={Logo} />
         </S.TitleText>
       </Link>
-      <S.NavMenu
-        selectedKeys={[pathname]}
-        items={[
-          {
-            key: 'github',
-            label: (
-              <a href={GITHUB_URL} target="_blank" data-cy="github-link">
-                GitHub
-              </a>
-            ),
-          },
-          {
-            key: 'docs',
-            label: (
-              <a href={DOCUMENTATION_URL} target="_blank" data-cy="documentation-link">
-                Documentation
-              </a>
-            ),
-          },
-          {
-            key: 'SubMenu',
-            label: <QuestionCircleOutlined data-cy="onboarding-link" style={{color: '#61175e', fontSize: 16}} />,
-            children: [
-              {
-                key: 'Onboarding',
-                label: (
-                  <a key="guidedTour" onClick={handleGuidedTourCLick}>
-                    Show Onboarding
-                  </a>
-                ),
-              },
-            ],
-          },
-        ]}
+      <HeaderMenu
+        onGuidedTourClick={onGuidedTourClick}
+        visible={visible}
+        setIsOpen={setIsOpen}
+        setVisible={setVisible}
+        pathname={pathname}
+        title={() => <Typography.Title level={5}>Take a quick tour of Tracetest?</Typography.Title>}
```

Comment:
This `title` prop is a little bit confusing here. It's added at the `HeaderMenu` level but this seems to be the Onboarding Confirmation screen title. 🤔

Reply:
Same for `setIsOpen`, `setIsVisible`. Not clear what is going to open or make it visible.

Reply:
solved